.github/workflows/databases.yml

```diff
@@ -140,7 +140,7 @@ jobs:
       - name: Execute tests
         run: vendor/bin/phpunit tests/Integration/Database
         env:
-          DB_CONNECTION: mysql
+          DB_CONNECTION: mariadb
```

Comment:
@taylorotwell these are the mariadb tests and previously it used the mysql connection. Now it uses its dedicated mariadb connection which this PR creates.

Reply:
So that the MariaDB Database tests are run against the MariaDB configuration instead of the MySQL configuration. Because of the changed collation in the mysql configuration which does not exist in MariaDB, this would throw an error.